app/controllers/users_controller.rb

```diff
@@ -24,9 +35,38 @@ def create
     end
   end
 
+  def edit; end
+
+  def update
+    if @user.update(user_params)
```

Comment:
```suggestion
    if @user.update user_params
```
update khác gì vs update_attributes

Reply:
cái này hôm chị vừa nói xong
-update: update 1 hoặc nhiều thuộc tính,validate, callback khi cập nhật. nếu thành công trả về true , nếu k false. có thêm dấu !  "update!" nó rẻ bắn exception nếu cập nhận false
-update_atrributes: vì có thêm "s" nên nó gần giống update. update_atrributes là 1 instance method. nói chung em tìm hiểu thì thấy nó giống hệt instance method: update. vẫn validate, và callback.
-còn update_attribute: chỉ cập nhật 1 thuộc tính. bỏ qua validate,callbak. trả về true,false
mấy cái này mà lâu k dùng đến là quên chị ạ. 
nếu em nói sai chỗ nào chị chỉ rõ và giải thích giúp em ạ.